server/controllers/crm/crm5Controller.ts

```diff
@@ -16,12 +16,32 @@ export default class Crm5Controller {
       const crm5Response = await this.crm5Service.getCrm(usn)
       const navigation = this.crmDisplayService.getCrmNavigation('crm5', usn, sectionId, crm5Response)
       const section = this.crmDisplayService.getCrmSection('crm5', sectionId, crm5Response)
+
+      const history = req.session.history || []
+      const currentUrl = `/crm5/${usn}/${sectionId || 'general-information'}`
+
+      if (req.query.fromBack) {
```

Comment:
It would be good to have some test cases in the crm5Controller.test.ts to cover these scenarios.